sample-plugin/javascript/src/index.js

```diff
@@ -0,0 +1,37 @@
+const Internal = require("./w3")
+const { Query } = require("./query")
+const { Mutation } = require("./mutation")
+
+class JsExamplePlugin extends Internal.JsExamplePlugin {
+  constructor(config) {
+    super({
+      query: config
```

Comment:
I saw that the configs aren't being used, so passing undefined is fine, but just for demonstration purposes it might make sense to include the mutation config here